src/github.js

```diff
@@ -96,25 +100,57 @@ async function fetch_changed_files() {
   return changed_files;
 }
 
-async function fetch_current_reviewers() {
+async function fetch_reviewers() {
   const context = get_context();
   const octokit = get_octokit();
 
-  const current_reviewers = [];
+  const reviewers = [];
+  const per_page = 100;
 
-  // API docs
-  // Generated octokit methods: https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/main/src/generated/endpoints.ts
-  // Available Rest APIs: https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#get-all-requested-reviewers-for-a-pull-request
-  const { data: response_body } = await octokit.pulls.listRequestedReviewers({
-    owner: context.repo.owner,
-    repo: context.repo.repo,
-    pull_number: context.payload.pull_request.number,
-  });
+  // GraphQL Docs: https://docs.github.com/en/graphql/reference/unions#pullrequesttimelineitems
+  // Pagination: https://github.com/octokit/plugin-paginate-graphql.js/?tab=readme-ov-file#usage
+  const response = await octokit.graphql.paginate(
+    `
+    query paginate($cursor: String, $repo: String!, $owner: String!, $number: Int!, $per_page: Int!) {
+      repository(owner: $owner, name: $repo) {
+          pullRequest(number: $number) {
+              timelineItems(first: $per_page, after: $cursor, itemTypes: REVIEW_REQUESTED_EVENT) {
+                  nodes {
+                      ... on ReviewRequestedEvent {
+                          requestedReviewer {
+                            ... on User {
+                                  login
+                            }
+                            ... on Team {
+                                  slug
+                            }
+                          }
+                      }
+                  }
+                  pageInfo {
+                      hasNextPage
+                      endCursor
+                  }
+              }
+          }
+      }
+  }`,
+    {
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+      number: context.payload.pull_request.number,
+      per_page: per_page,
+    }
+  );
 
-  current_reviewers.push(...response_body.users.map((user) => user.login));
-  current_reviewers.push(...response_body.teams.map((team) => 'team:'.concat(team.slug)));
+  reviewers.push(...(response?.repository?.pullRequest?.timelineItems?.nodes || []).map((reviewer) => {
+    if (reviewer?.requestedReviewer?.slug) {
+      return 'team:'.concat(reviewer.requestedReviewer.slug);
+    }
+    return reviewer?.requestedReviewer?.login || '';
+  }).filter((reviewer) => reviewer.length));
```

Comment:
```suggestion
  let reviewerNodes = response?.repository?.pullRequest?.timelineItems?.nodes || [];
  for (node in reviewerNodes) {
    if (node?.requestedReviewer?.slug) {
        reviewers.push('team:'.concat(node.requestedReviewer.slug));
    } else if (node?.requestedReviewer?.login)  {
      reviewers.push(node.requestedReviewer.login);    
    }
  }
```

Instead of doing a map and then filtering it down in order to populate the `reviewers` array, I think it might be more readable to instead use a for loop like something in the suggestion, since that more directly encodes "we will maybe add a given reviewer entry to the array".